docs/editors/emacs.md

```diff
@@ -74,6 +77,23 @@ To use Metals in Emacs, place this snippet in your Emacs configuration (for exam
 
 ;; Add company-lsp backend for metals
 (use-package company-lsp)
+
+;; Use the Debug Adapter Protocol for running tests and debugging
```

Comment:
You can find more details on DAP at its [github project](https://github.com/emacs-lsp/dap-mode)